src/internal/observable/iif.ts

```diff
@@ -90,8 +90,8 @@ import { SubscribableOrPromise } from '../types';
  */
 export function iif<T = never, F = never>(
```

Comment:
Can you remove the ` = never` bit?

Reply:
Let's tackle this in another pass. There's a lot of about `iif`'s type that I think is wrong.